rpcd/playbooks/roles/filebeat/defaults/main.yml

```diff
@@ -154,6 +154,13 @@ filebeat_logging_paths:
       pattern: "{{ multiline_openstack_pattern }}"
       negate: 'true'
       match: after
+  - paths:
```

Comment:
Also, as a separate work item, we should probably rework how we put down filebeat files so that they can be disabled if unnecessary.

Either way, since there's no path to these log files there should be little problem to this being laid down everywhere.